src/translations/aggrid-locale-fr.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+const aggrid_locale_fr = {
+    'grid.contains': 'Contient',
+    'grid.notContains': 'Ne contient pas',
+    'grid.equals': 'Égal à',
+    'grid.notEqual': "N'est pas égal à",
+    'grid.startsWith': 'Commence par',
+    'grid.endsWith': 'Finit par',
+    'grid.blank': 'Vide',
+    'grid.notBlank': "N'est pas vide",
+    'grid.inRange': 'Intervalle',
+    'grid.lessThan': 'Moins que',
+    'grid.lessThanOrEqual': 'Moins ou égal à',
+    'grid.greaterThan': 'Plus que',
+    'grid.greaterThanOrEqual': 'Plus ou égal à',
+    'grid.andCondition': 'ET',
+    'grid.orCondition': 'OU',
+    'grid.inRangeStart': 'Début',
+    'grid.inRangeEnd': 'Fin',
+    'grid.filterOoo': 'Filtrer...',
+    'grid.noRowsToShow': 'Aucune Donnée',
+    'grid.noCalculation': 'Pas de calcul lancé',
+    'grid.failed': 'Echec du calcul',
+    'grid.running': 'Calcul en cours...',
+    'grid.noLimitViolation': 'Pas de contrainte',
+    'grid.page': 'Page',
+    'grid.to': 'à',
+    'grid.of': 'de',
+    'grid.firstPage': 'Première page',
+    'grid.previousPage': 'Page précédente',
+    'grid.nextPage': 'Page suivante',
+    'grid.lastPage': 'Dernière page',
+    'grid.ariaSortableColumn': 'Appuyez sur ENTRÉE pour trier.',
+    'grid.ariaMenuColumn':
+        'Appuyez sur CTRL+ENTRÉE pour ouvrir le menu de colonne.',
```

Comment:
'grid.ariaMenuColumn': 'Appuyez sur CTRL+ENTRÉE pour ouvrir le menu de colonne. ',

Reply:
There is no difference except the spacing after the sentence, which I will not add. If adjustments need to be made to space elements between them, this is a CSS issue, not i18n.

Reply:
no problem